packages/providers/src/transaction-summary/date.ts

```diff
@@ -0,0 +1,15 @@
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime.js'; // eslint-disable-line import/extensions
+import { TAI64 } from 'tai64';
+
+dayjs.extend(relativeTime);
+
+export const tai64toDate = (tai64Timestamp: string) => {
+  const timestamp = TAI64.fromString(tai64Timestamp, 10).toUnix();
+  return dayjs(timestamp * 1000);
+};
+
+export const formatDate = (tai64Timestamp: string) => tai64toDate(tai64Timestamp).fromNow();
```

Comment:
Instead of using `dayjs`, it's enough to just convert to ISO 8601 format and have this function's implementation be:
```ts
export function formatDate(tai64Timestamp: string) {
const unixTimestamp = TAI64.fromString(tai64Timestamp, 10).toUnix();
const date = new Date(unixTimestamp * 1000);
return date.toISOString();
}
```

We can then rename `timeFormatted` into `timeISO` and we're good to go. There's no need depend on `dayjs`, or any other library for that matter, as it's sufficient to convert TAI64 into ISO 8601. Whoever wants to use the `timeISO` value can do so and they can use their favorite date library to do with it whatever they wish.

Reply:
Didn't see [Luiz's comment](https://github.com/FuelLabs/fuels-ts/pull/1252#pullrequestreview-1625093044). His suggestion is better.